app/src/helpers/index.ts

```diff
@@ -0,0 +1,35 @@
+import type { SupportedChainId } from '~/types'
+
+export const getEtherscanApiUrl = (chainId: SupportedChainId) => {
+  switch (chainId) {
+    case '1':
+      return 'https://api.etherscan.io'
+    case '5':
+      return 'https://api-goerli.etherscan.io'
+    case '11155111':
+      return 'https://api-sepolia.etherscan.io'
+    case '10':
+      return 'https://api-optimistic.etherscan.io'
+    default:
+      throw new Error('Not supported chainID')
+  }
+}
+
+export const getEtherscanApiKey = (chainId: SupportedChainId) => {
```

Comment:
Recommendation: combine those two into `getChainConfig(chainName: string)` to
- return object with `{ endpoint: 'http://...', apiKey: '...' }`
- use readable names for chain names instead of ids, eg `goerli`